### ISSUE TITLE
[Feature Request]  A conf option that allows to already select immediately after appending to the list of items.

### ISSUE THREAD

**Issue by cocoonkid** [erebe/greenclip#124]
[Feature Request]  A conf option that allows to already select immediately after appending to the list of items.

I request this because when working a lot with virtual machines I need to specifically select it from rofi to be able to copy/paste the item to the host machine in one go.

Happens in VMware workstaion and Virtualbox.

Thanks for this amazing application.

**Comment by TAforever** [comment]
Yes, it would be great if the developers add this feature so that the copied can be pasted even if the window from which the copy was made is closed without resorting to greenclip.

**Comment by pc00per** [comment]
You can select the recent item through xclip directly & paste it directly into the target. Makes even easier for you. [Here's my setup btw](https://github.com/erebe/greenclip/discussions/137#discussioncomment-3986454).

**Comment by PRESFIL** [comment]
This feature is called **force retaking of clipboard ownership**. Users have already asked
about it, but @erebe [said][said] that it is not possible now, unfortunately.

I came to `greenchip` from `clipmenu` for MIME support, but I lost **force retaking of clipboard
ownership**, sadly.

**Duplicate of:**

- #123

[said]: https://github.com/erebe/greenclip/issues/123#issuecomment-1094612925
